spec/controllers/dogs_controller_spec.rb

```diff
@@ -80,6 +80,13 @@
         expect(assigns(:dogs)).to match_array([adoptable_dog, adoption_pending_dog, coming_soon_dog])
       end
 
+      it 'with all dogs paramater set all dogs are returned' do
+        get :index, params: {all_dogs: true}, session: { mgr_view: true }
```

Comment:
Covering that at https://github.com/ophrescue/RescueRails/blob/2ebb0a19684708fe42be30247f369610ae65b1b7/spec/controllers/dogs_controller_spec.rb#L78 I think?